pkgs/applications/audio/real_time_config_quick_scan/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv, fetchFromGitHub, perlPackages, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  pname = "realTimeConfigQuickScan";
+  version = "unstable-2020-08-03";
+
+  src = fetchFromGitHub {
+    owner  = "raboof";
+    repo   = pname;
+    rev    = "4b482db17f8d8567ba0abf33459ceb5f756f088c";
+    sha256 = "00l69gzwla9gjv5kpklgxlwnl48wnh8h6w0k8i69qr2cxigg4rhj";
+  };
+
+  buildInputs = [ perlPackages.perl makeWrapper ];
+
+  dontBuild = true;
+
+  installPhase = ''
+    mkdir -p $out/bin
+    mkdir -p $out/share/doc
+    # Install Script Files:
+    # *.pm files
+    for i in *.pm; do
+    install -Dm 755 "$i" "$out/share/$i"
+    done
+    # Install doc files:
+    install -D COPYING  "$out/share/doc/COPYING"
```

Comment:
This can create a conflict with another package installing the same file. It should go to `$out/share/doc/$name` or something.